throughput/README.md

```diff
@@ -0,0 +1,96 @@
+Throughput Reporting Sink Module
+================================
+
+Is a sink that counts received messages and periodically report the witnessed thoughput. This is to provide
+a method for users to identify the performance of XD in different environments and deployment
+types.
+
+## Requirements
+
+In order to install the module and run it in your Spring XD installation, you will need to have installed:
+
+* Spring XD version 1.1.x ([Instructions](http://docs.spring.io/spring-xd/docs/current/reference/html/#getting-started))
+
+## Building with Maven
+
+	$ mvn package
+
+The project's [pom](pom.xml) declares `spring-xd-module-parent` as its parent. This adds the dependencies needed to test the module and also configures the [Spring Boot Maven Plugin](http://docs.spring.io/spring-boot/docs/current/reference/html/build-tool-plugins-maven-plugin.html) to package the module as an uber-jar, packaging any dependencies that are not already provided by the Spring XD container. See the [Modules](http://docs.spring.io/spring-xd/docs/current/reference/html/#modules) section in the Spring XD Reference for a more detailed explanation of module class loading.
+
+## Building with Gradle
+
+	$./gradlew clean test bootRepackage
```

Comment:
Because of copy/paste.
Indeed, if build encompasses bootRepackage, this is better